lib/dpl/provider/pypi.rb

```diff
@@ -32,23 +32,15 @@ def skip_upload_docs?
           (options.has_key?(:skip_upload_docs) && options[:skip_upload_docs])
       end
 
-      def self.install_setuptools
-        shell 'wget https://bootstrap.pypa.io/ez_setup.py -O - | sudo python'
-        shell 'rm -f setuptools-*.zip'
-      end
-
-      def self.install_twine
-        shell("pip install twine", retry: true) if `which twine`.chop.empty?
-      end
-
-      def initialize(*args)
-        super(*args)
-        self.class.pip 'wheel' if pypi_distributions.to_s.include? 'bdist_wheel'
+      def install_deploy_dependencies
+        unless context.shell "wget https://bootstrap.pypa.io/get-pip.py && " \
```

Comment:
@jezdez 
instead of storing python script in the fs you could use pipe:
```shell
wget -O - https://bootstrap.pypa.io/get-pip.py | python - --no-setuptools --no-wheel
```

Reply:
Ok. I think we've optimized this patch enough 😁